conda.recipe/meta.yaml

```diff
@@ -3,10 +3,9 @@
 {% set license = data.get('license') %}
 {% set description = data.get('description') %}
 {% set url = data.get('url') %}
-# this will reproduce the version from tags to match versioningit
-{% set raw_version = environ.get('GIT_DESCRIBE_TAG', '0.0.0') %}
-{% set version = raw_version.lstrip('v') %}
-{% set version_number = environ.get('GIT_DESCRIBE_NUMBER', '0') | string %}
+# this will get the version set by environment variable
+{% set version = environ.get('VERSION') %}
+{% set version_number = version.split('+')[0] %}
```

Comment:
Please add the build number like in the [python project template](https://github.com/neutrons/python_project_template/blob/main/conda.recipe/meta.yaml#L11-L12)